### PR TITLE
Added footprint for TPS6303x.

### PR DESCRIPTION
**Pull request by cnieves1**
Datasheet: http://www.ti.com/lit/gpn/tps63030

### CODE REVIEW

**Comment by chschlue** [comment]
#422

**Comment by cnieves1** [comment]
Could you please tell me what exact changes are required by https://github.com/pointhi/kicad-footprint-generator/pull/422 ?

As I understand, it is a change in the script. 
Do I need to change any data introduced in this PR?

Thanks,

**Comment by chschlue** [comment]
see https://github.com/pointhi/kicad-footprint-generator/pull/452#issuecomment-579143412

**Comment by cnieves1** [comment]
Footprint regenerated at:
  https://github.com/KiCad/kicad-footprints/pull/2012

Thanks!

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/wson.yaml [hunk 1]
```diff
@@ -281,6 +281,64 @@ Texas_S-PWSON-N8:
...
+  #custom_name_format: 'Texas_S-PWSON-N{pincount}_EP{ep_size_x:g}x{ep_size_y:g}mm{vias:s}'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    minimum: 2.4
+    maximum: 2.6
+  body_size_y:
+    minimum: 2.4
+    maximum: 2.6
+  overall_height:
+    minimum: 0.6
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/wson.yaml]: Is min height given anywhere?

**Comment by chschlue** [comment]
FP has been merged.

**Pull request merged**